soh/src/boot/z_std_dma.c

```diff
@@ -1,5 +1,7 @@
 #include "global.h"
 #include "vt.h"
+#include <stdio.h>
+#include <assert.h>
```

Comment:
Would it make more sense to include assert.h once in say, globals.h, as opposed to including it in a bunch of individual files, or would that not work?

Reply:
Louis updated me on Discord, just wants it to be included where it's needed instead of in a common header. Which is fair enough.